saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -302,7 +310,12 @@ def perform_mutation(cls, _root, info, **data):
                     lines_to_add.append((quantity, variant))
             else:
                 raise ValidationError(
-                    {"quantity": "Ensure this value is greater than or equal to 1."}
+                    {
+                        "quantity": ValidationError(
+                            "Ensure this value is greater than or equal to 1.",
+                            code=OrderErrorCode.QUANTITY_LESS_THAN_ONE,
```

Comment:
We have at least these codes that are similar: `POSITIVE_NUMBER_REQUIRED`, QUANTITY_LESS_THAN_ONE`. I think that instead, we should have a few generic ones such as `LESS_THAN`, `LESS_THAN_OR_EQUAL_TO` and the same for "greater" checks.